### ISSUE TITLE
Electron App + macOS Ventura: Firewall Prompt at Every Launch

### ISSUE THREAD

**Issue by avagdumortesin** [foundryvtt/foundryvtt#8959]
### What happened?

Every launch of the FoundryVTT Electron App on macOS Ventura prompts to allow or deny access to the network for Foundry, even though it is added to the firewall settings.

### What ways of accessing Foundry can you encounter this issue in?

- [X] Native App (Electron)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

M1 Mac mini
macOS Ventura Public Beta 13.3 (22E5219e)
FoundryVTT Electron app 10.288.0

Ensure Firewall is enabled on macOS.
Launch the FoundryVTT Electron app; allow firewall access.
Quit and relaunch the Electron app; notice prompt appears again.

### What core version are you reporting this for?

Version 10 291

### Bug Checklist

- [X] The issue occurs while all Modules are disabled

**Comment by avagdumortesin** [comment]
Link to screen recording of the behavior:
[https://www.dropbox.com/s/wdf33948w43nfgq/Screen%20Recording%202023-02-27%20at%202.35.18%20PM.mov?dl=0](https://www.dropbox.com/s/wdf33948w43nfgq/Screen%20Recording%202023-02-27%20at%202.35.18%20PM.mov?dl=0)

**Comment by aaclayton** [comment]
Confirmed this behavior is occuring. It probably has to do with the electron version included in V10 (Electron v20) not properly handling this prompt in new MacOS. Our next major version V11 upgrades to Electron v23. We will check and see if this issue is observed there or not.

**Comment by unsoluble** [comment]
Can confirm this does continue to happen in 11.298 with Electron 23.2.3 (under Ventura 13.1).

**Comment by avagdumortesin** [comment]
Confirmed; downloaded 11.298, replaced the previous version, and launched twice. First launch prompted for microphone and camera access, as well as firewall access; second launch only prompted for firewall access.
Ventura 13.4

**Comment by aaclayton** [comment]
That's unfortunate. I'm not sure what we can do differently.

**Comment by unsoluble** [comment]
It's definitely something to do with the Electron packaging that causes this (isn't endemic with other app bundles requesting port access under Ventura, nor is it an issue with Foundry running as a Node process).

**Comment by avagdumortesin** [comment]
Yeah, to my knowledge, Foundry is one of three Electron apps I have installed (Slack and Discord being the others). Since neither of them do any sort of hosting, I'm not sure what else I can test that might allow a bug report to Electron.

**Comment by aaclayton** [comment]
Might require https://github.com/foundryvtt/foundryvtt/issues/6051
